test/send-to-fluentd/20000-fake-1-logging-tcp.py

```diff
@@ -0,0 +1,21 @@
+import testlib
+import time
+
+testlib.send_logs.tcp_connect('localhost', 20000)
+
+# ts = testlib.timeutils.str2ts('2020-01-02T10:40:00.000Z')
+ts = int(time.time())
+
+# s1 = '2020-01-02T18:31:00.000Z'
+# t1 = testlib.timeutils.str2ts(s1)
+# s2 = testlib.timeutils.ts2str(t1)
+
+log = testlib.mock_data.mock_logging_data_20001(ts, 1)
```

Comment:
mock_logging_data_20001 doesn't match the 20000 port. Shall we change 20000 to 20001 in `testlib.send_logs.tcp_connect('localhost', 20000)` and the filename?

Reply:
Get it.